getting-started.md

```diff
@@ -19,6 +19,11 @@ You need to install `yq`. You can do this like so:
 ```
 go install github.com/mikefarah/yq/v4@latest
 ```
+You also need [ko](https://gochronicles.com/containertools-ko/) a tool for building lighter, more secure container images.
```

Comment:
Could we link to https://ko.build/ instead of that blog post?

Reply:
@bobcallaway I need approval for the workflows to run. Is this something you would do?
cc @znewman01